### PR TITLE
plugins: add plugin for zoxide

### PR DESCRIPTION
**Pull request by ajeetdsouza**
[zoxide](https://github.com/ajeetdsouza/zoxide) a smart, fast, and configurable alternative to your cd command.

### CODE REVIEW

**Comment by ajeetdsouza** [comment]
@nntoan @Kreyren I'd appreciate a review on this when you can spare the time.

**Review by nntoan — APPROVED** [review]
LGTM.

**Comment by ajeetdsouza** [comment]
@nntoan can this be merged?

File: plugins/zoxide/zoxide.plugin.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Check if zoxide is installed
+if [ -x "$(command -v zoxide)" ]; then
```
Comment (akinomyoga) [inline plugins/zoxide/zoxide.plugin.sh:4]:
```suggestion
if type_exists zoxide; then
```

**Comment by ajeetdsouza** [comment]
@akinomyoga thanks for the suggestion. Can this be merged now?

**Comment by akinomyoga** [comment]
Yes, thank you for responding to the suggestion! LGTM, I'll merge now.

**Comment by akinomyoga** [comment]
@ajeetdsouza Thank you again for submitting the PR and for quick responses to the comment!

**Pull request merged**